MavenQuickStart/quickApp/src/main/java/com/globant/topicthree/exerciseone/ShoppingCart.java

```diff
@@ -0,0 +1,56 @@
+package com.globant.topicthree.exerciseone;
+
+import java.io.Serializable;
+import java.util.LinkedList;
+import java.util.List;
+import java.util.UUID;
+
+/**
+ * This entity represent a virtual Shopping Cart. of this.
+ * 
+ * @author andres.vaninetti
+ *
+ */
+public class ShoppingCart implements Serializable {
+
+	private static final long serialVersionUID = 1L;
+
+	private List<Product> products;
+
+	private String id;
+
+	/**
+	 * Default constructor.
+	 * 
+	 */
+	public ShoppingCart() {
+		this.products = new LinkedList<Product>();
```

Comment:
For this example any list implementation will work. But take a look at how are implemented, because sometimes you have to choose between ArrayList/LinkedList/etc to check which one is a better choice for your solution. Read it from the official documentation and also take a look at links like this one: https://stackoverflow.com/questions/322715/when-to-use-linkedlist-over-arraylist